packages/fxa-payments-server/package.json

```diff
@@ -23,7 +23,7 @@
     "format": "prettier --write --config ../../_dev/.prettierrc '**'",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "NODE_ENV=production npm run build-css && build-storybook && cp -r public/images storybook-static/",
-    "merge-ftl": "grunt merge-ftl",
+    "merge-ftl": "grunt merge-ftl && yarn l10n:bundle fxa-payments-server payments",
```

Comment:
I think this needs `payments.ftl` _and_ `react.ftl`, but not branding yet.

Reply:
I'd be happy to change this, but the grunt file just had payments... and a hard coded branding file? https://github.com/mozilla/fxa/blob/b821a6a6ea6fc8f6a23b688f0ae914daca775136/packages/fxa-payments-server/Gruntfile.js#L11

Reply:
Yeah, payments has its own branding file temporarily until we can combine them (there's a ticket in the L10n Improvements epic). I was going by what we were passing into the AppLocalizationProvider [previously](https://github.com/mozilla/fxa/pull/15018/files#diff-fa3755a85644684faf5a3d48ec13dbc50e7e5fa23d71c7e26ef80ffd87b53044L98) and if you check SubPlat staging/prod you'll see the two requests.

Reply:
Ah gotcha. I'll add react. You've convinced me 👍